web-api-project/Project.py

```diff
@@ -0,0 +1,68 @@
+#lets you interact with web api
+import requests
+
+
+#gets the info from the api to to answer the country input
+def Country_info(country):
+    Base_url = "https://restcountries.com/v3.1/name/" + country
+    req = requests.get(Base_url)
+    info = req.json()
+    return info
+
+
+#this should show up and prompt you to put in a country
+def begin():
+    defined_country = input("Put a country: ")
+   
+    #attempts to try and run the info and if not, the except steps into play.
+    try:
+        info = Country_info(defined_country)
+    except Exception as e:
+        print("None of this information can be found.")
+        return
+
+
+    #This grabs information about the places
+    data = info[0]
+    capital_city = data.get('capital', ['No capital'])[0]
+    Population_of_country = data.get('population', "This is not known")
+    Place = data.get('place', "This isn't made clear")
```

Comment:
`place` is not a valid field.
https://gitlab.com/restcountries/restcountries/-/blob/master/FIELDS.md